src/tempestApi.ts

```diff
@@ -101,4 +118,52 @@ export class TempestApi {
 
   }
 
+  public async getTempestBatteryLevel(): Promise<number> {
```

Comment:
```suggestion
public async getTempestBatteryLevel(device_id: number): Promise<number> {
```

Reply:
`getStationCurrentObservation`, `getTempestBatteryLevel` and `getTempestDeviceId` each make use of different API calls to obtain the required information. 

As a result, the `getTempestDeviceId` method is required. Understand that as the tempest_device_id value is static and should only be called once when plugin is initialized.  Additional updates will be required and will incorporate in v2.1.8 (other changes are coming).